lib-ee/emqx_ee_bridge/src/emqx_ee_bridge.erl

```diff
@@ -183,3 +187,18 @@ pgsql_structs() ->
             {matrix, <<"Matrix">>}
         ]
     ].
+
+clickhouse_structs() ->
+    [
+        {Type,
+            mk(
+                hoconsc:map(name, ref(emqx_ee_bridge_clickhouse, "config")),
+                #{
+                    desc => <<Name/binary, "Clickhouse Bridge Config">>,
+                    required => false
+                }
+            )}
+     || {Type, Name} <- [
+            {clickhouse, <<"Clickhouse">>}
```

Comment:
not necessary to use a tuple list here ?

Reply:
Thanks, fixed in latest commit.